dist/lob-api-bundled.yml

```diff
@@ -6362,6 +6362,9 @@ components:
             tracking_events:
               description: Tracking events are not populated for registered letters.
               type: array
+              items:
+                $ref: '#/components/schemas/tracking_event_normal'
+                example: []
```

Comment:
@Kilo59 why is the example an empty array?

Reply:
The example already has an empty array (examples **should not** generally be included in models).

```
    tracking_number: null
    tracking_events: []
```
from `responses/letter`.